quickstarts/languages/nodejs/index.js

```diff
@@ -1,3 +1,18 @@
+// Copyright 2019 Google LLC
```

Comment:
**Praise:** You even dug up the correct year of creation (of each file). It's a nit, but appreciate the attention to detail!